templates/CRM/Contact/Form/Merge.tpl

```diff
@@ -273,6 +271,25 @@
     CRM.$( "#main_" + blockname + "_" + blockId + "_overwrite" ).html( label );
   }
 
+  // Find a matching main contact location block by entity, location and type
```

Comment:
I'm not quite sure what our style spec is on js  - for php it should be a 'proper' function comment block & I suspect the same for js